R/mplot.R

```diff
@@ -0,0 +1,94 @@
+#' mplot
+#'
+#' Get a quick visualization of the content of most magpie objects.
+#'
+#' @param px The magpie object to be visualized
+#' @param global Whether data should be aggregated over regions to global values (by default TRUE).
+#' @param total Whether the total of all data values should also be visualized (by default TRUE).
```

Comment:
The default values are shown in the documentation anyway, no need to confuse anyone when this is out of sync :wink: 
```suggestion
#' @param global Whether data should be aggregated over regions to global values.
#' @param total Whether the total of all data values should also be visualized.
```

Reply:
ups good to know :smile: I might have done that repeatedly in recent commits.